tests/system/action/tag/test_create.py

```diff
@@ -35,3 +35,12 @@ def test_create_wrong_field(self) -> None:
             "data must not contain {'wrong_field'} properties",
             response.json["message"],
         )
+
+    def test_create_no_permissions(self) -> None:
+        self.set_management_level("can_manage_users")
+        self.create_model("meeting/577", {"name": "name_YBEqrXqz"})
+        response = self.request(
+            "tag.create", {"name": "test_Xcdfgee", "meeting_id": 577}
+        )
+        self.assert_status_code(response, 403)
+        self.assert_model_not_exists("tag/1")
```

Comment:
I would like to have in general an self.assertIn with the message to learn and understand and notice, if there is another kind of error as expected. In this case it is "You do not belong to meeting 577".

Reply:
done.